lib/msal-browser/docs/iframe-usage.md

```diff
@@ -0,0 +1,140 @@
+# Using MSAL in iframed apps
+
+By default, MSAL prevents full-frame redirects to **Azure AD** login UX when an app is rendered inside an iframe, which means you cannot use [redirect APIs](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#redirect-apis) for user interaction with the IdP:
+
+- This restriction is imposed since **Azure AD** will refuse to render prompts for **credential entry** and **consent** in an iframe by throwing the `X-FRAME OPTIONS SET TO DENY` error (see: [x-frame options](https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-x-frame-options-header)), a measure taken to prevent [clickjacking attacks](https://owasp.org/www-community/attacks/Clickjacking).
```

Comment:
It will also refuse to render logout prompts.

Reply:
And thus I would add a section on using `logoutPopup`.

Reply:
@jasonnutter add this bullet point:

> - Similarly, you'll have to use the [logoutPopup()](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logout.md#logoutpopup) API for sign-outs (:warning: if your app is using a version of msal-browser older than v2.13, make sure to upgrade and replace the `logout()` API, as it will attempt a full-frame redirect to Azure AD).